flow360/component/simulation/operating_condition/operating_condition.py

```diff
@@ -202,19 +202,20 @@ class GenericReferenceCondition(MultiConstructorBaseModel):
     ====
     """
 
+    private_attribute_input_cache: GenericReferenceConditionCache = GenericReferenceConditionCache()
```

Comment:
how does it impact type hints? I think IDE will list the options in order of definition, thus private_... and type_name should be last?

Reply:
You are right. input_cache will become the first.

Reply:
Reverted the ordering of private_attribute_input_cache to make type hint a bit cleaner.